palestirnative/palestirnative
# feat: add bdnaash + similar websites logos + make social in new tab

## Description

This PR:
Closes #21 
Closes #23 
Closes #24 

### How to test it

1. Go to the footer
2. You should see the logos alongside the similar websites
3. You should see "Bdnaash" in the list
4. Click on the LinkedIn icon, the page should open in a new tab

### Screenshots

![image](https://github.com/palestirnative/palestirnative/assets/10698248/4f43d1da-d865-4300-b906-6f14ae3e2e37)

## Approach

Added an image w-5 for the logos and made the similar websites links flexboxes
